neqo-http3/src/server.rs

```diff
@@ -960,6 +970,16 @@ mod tests {
         assert_eq!(stop_sending, 1);
     }
 
+    /*
+    #[test]
+    fn test_client_priority_update() {
+        let (mut hconn, mut peer_conn) = connect();
+        let client = Http3ClientEvent::new()
+        let stream_id = peer_conn.stream_create(StreamType::BiDi).unwrap();
+        peer_conn.stream_send(stream_id, REQUEST_WITH_BODY).unwrap();
+    }
+    */
```

Comment:
Do you need this test?

Reply:
Yeah, forgot to delete it. It got replaced by `neqo-http3/tests/priority.rs`